src/main/resources/com/microsoft/jenkins/kubernetes/KubernetesDeployContext/config.properties

```diff
@@ -18,3 +18,5 @@ enableConfigSubstitution_title = Enable Variable Substitution in Config
 dockerCredentialsSection_title = Docker Container Registry Credentials / Kubernetes Secrets
 secretName_title = Secret Name
 dockerCredentials_title = Docker Container Registry Credentials
+
+deleteResource_title = Delete Resources.
```

Comment:
Full stop is unnecessary here.

Reply:
I don't understand what it means. Can you explain it more?

Reply:
We don't need the `.` at the end 😄 .

Reply:
fix